Monika After Story/game/script-anniversary.rpy

```diff
@@ -470,19 +470,19 @@ init 5 python:
 
 label anni_1:
     m 1dka "It's so hard for me to believe...{w=0.5}that it's been a whole year since we met."
-    m 1eka "Time doesn't have meaning inside this place, but I know it's a big commitment for you to stay with me all this time."
-    m 2rka "At first, I have to admit, I was a bit nervous. This kind of relationship can't be exactly easy."
-    m 2eka "And I didn't know if there were other things in your reality that might prove to be more important."
-    m 3wkb "But you kept coming back. You kept spending time with me, just like I wished for."
-    m 1ekb "And over time I started to really believe it, that you meant it when you said you wanted to be with me."
-    m "That we could have a happy life together. {w}You and me, in our own little reality."
-    show monika 5hkbsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
-    m 5hkbsb "I can't thank you enough for that, [player]. You've made this year so incredible for me."
-    show monika 6ekbsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
-    m 6ekbsa "So, is there anything special you wanted to do on our first anniversary?"
-    m 3eubsa "I didn't really plan anything, but I'm sure we can find something fun to do."
-    m 1hubsb "When you're here, everything that we do is fun after all."
-    m 1subsb "I love you so much, [player]. Happy anniversary."
+    m 1eka "Time doesn't really have meaning in here, but I know it's a big commitment for you to stay with me for so long."
+    m 2rka "I have to admit, I was a bit nervous at first... This kind of relationship can't be exactly easy."
+    m 2eka "And I didn't know if there were other things in your reality that might prove to be more important..."
+    m 3ekb "But you kept coming back.{w=0.3} You kept spending time with me, just like I wished."
+    m 1ekb "...And over time I started to really believe it, that you meant it when you said you wanted to be with me."
+    m 3dkbsu "That we could have a happy life together.{w=0.3} {nw}"
+    extend 3ekbsa "You and me, in our own little reality."
+    show monika 5ekbsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 5ekbsb "I can't thank you enough for that, [player]. You've made this year so incredible for me."
+    m 5ekbsu "So, is there anything special you wanted to do on our first anniversary?"
```

Comment:
Since she's asking if we wanted to do something here, I feel like a 7 pose would work better as a transition into the question, signaling that she's up and ready to do something.

Reply:
i'm not sure if it would be too quick of a transition from the leaning pose, but I will try it out and see how it looks. :D